### PR TITLE
Cast params

### LINKED ISSUE #9: ThrottlingMiddleware cannot accept int
When using `ThrottlingMiddleware` with `throttle:x,x` the call to `getHeaders` fails because it's expecting an integer.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Fixes #9

### CODE REVIEW

File: src/ThrottlingMiddleware.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ protected function buildException(string $key, int $limit, bool $headers)
 
         $exception = new ThrottlingException($after, 'Rate limit exceeded.');
 
-        $headers = $this->getHeaders($key, $limit, $headers, $after, $exception->getHeaders());
+        $headers = $this->getHeaders($key, (int) $limit, $headers, $after, $exception->getHeaders());
```
Comment (GrahamCampbell) [inline src/ThrottlingMiddleware.php]: Limit is already an integer, since it came in via an argument to buildException. That cast won't do anything?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/ThrottlingMiddleware.php (hunk 1 above)
Comment (jbrooksuk) [reply src/ThrottlingMiddleware.php]: Pretty sure when I was testing this, I had to cast it for it to work.

Comment (jbrooksuk) [reply src/ThrottlingMiddleware.php]: You're right, I made too many changes. Will push again.

**Comment by GrahamCampbell** [comment]
There are other cases that need fixing still I think. I'll take a look at the weekend.

**Comment by GrahamCampbell** [comment]
Fixed with alternate solution.

**Pull request closed without merging**